src/com/sun/ts/tests/signaturetest/signature-repository/jakarta.servlet.jsp.sig_3.0_se8

```diff
@@ -1,425 +1,10 @@
 #Signature file v4.3
-#Version 2.3_se8
+#Version 3.0_se8
```

Comment:
Hi Mark - This sig file has issues and is being regenerated in https://github.com/eclipse-ee4j/jakartaee-tck/pull/301 to fix . Can you please delete this file.

Reply:
I'm assuming you mean from the PR. Will do.